templates/default/pushgateway.service.erb

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description = Prometheus  
+After = syslog.target network.target remote-fs.target
```

Comment:
Is there any dependency of prometheus on pushgateway?

Reply:
No, the entire architecture is made to be independent. Prometheus keeps scraping the targets, no matter if they are available or not.